src/frontend/komponenter/Fagsak/Dokumentutsending/DokumentutsendingSkjema.tsx

```diff
@@ -54,6 +57,10 @@ const SendBrevKnapp = styled(Button)`
     margin-right: 1rem;
 `;
 
+const StyledBrevmottakereAlert = styled(BrevmottakereAlert)<BrevmottakereAlertFagsakProps>`
```

Comment:
Vi trenger ikke å ta med typene til props her dersom vi ikke bruker komponentens props for å interpolere styling 🙂